### ISSUE TITLE
Cannot install diyHue on latest HomeAssistant OS

### ISSUE THREAD

**Issue by Bx16** [diyhue/diyHue#918]
### Issue does not already exist?

I have searched and found no existing issue

### Select Environment

Install Script

### Home Assistant related?

Yes

### Description

Cannot install the diyHUE addon in Home Assistant as an error pops up

### Errorlog:

Logger: homeassistant.components.hassio
Source: components/hassio/websocket_api.py:130
Integration: Home Assistant Supervisor ([documentation](https://www.home-assistant.io/integrations/hassio), [issues](https://github.com/home-assistant/core/issues?q=is%3Aissue+is%3Aopen+label%3A%22integration%3A+hassio%22))
First occurred: 17:53:17 (3 occurrences)
Last logged: 17:58:55
Failed to to call /addons/c863ef44_diyhue/install - The command '/bin/bash -o pipefail -c curl -J -L -o $diyhue.tar.gz "https://github.com/diyhue/diyHue/archive/refs/heads/master.tar.gz" && tar xzvf $diyhue.tar.gz --strip-components=1 -C diyhue && rm -rf $diyhue.tar.gz && pip3 install -r /diyhue/requirements.txt --no-cache-dir' returned a non-zero code: 1

### Steps to reproduce

Just try to install

### Please enter your operating system details here

HAOS Linux 6.1.34

### What DiyHue version(branch) are you using?

master (latest)

**Comment by tino0815** [comment]
I get the same error:

Errorlog:
Logger: homeassistant.components.hassio
Source: components/hassio/websocket_api.py:130
Integration: Home Assistant Supervisor (documentation, issues)
First occurred: 19. Juli 2023 um 11:18:18 (9 occurrences)
Last logged: 20. Juli 2023 um 23:33:35

Failed to to call /addons/c863ef44_diyhue/install - The command '/bin/bash -o pipefail -c curl -J -L -o $diyhue.tar.gz "https://github.com/diyhue/diyHue/archive/refs/heads/master.tar.gz" && tar xzvf $diyhue.tar.gz --strip-components=1 -C diyhue && rm -rf $diyhue.tar.gz && pip3 install -r /diyhue/requirements.txt --no-cache-dir' returned a non-zero code: 1
Failed to to call /store/repositories - Cmd('git') failed due to: exit code(128) cmdline: git clone -v --recursive --depth=1 --shallow-submodules -- https://github.com/diyhue/hassio-addon/tree/master/diyhue /data/addons/git/81a68243 stderr: 'Cloning into '/data/addons/git/81a68243'... fatal: repository 'https://github.com/diyhue/hassio-addon/tree/master/diyhue/' not found '
Failed to to call /addons/238a16ee_diyhue/install - The command '/bin/bash -o pipefail -c curl -J -L -o $diyhue.tar.gz "https://github.com/diyhue/diyHue/archive/refs/heads/master.tar.gz" && tar xzvf $diyhue.tar.gz --strip-components=1 -C diyhue && rm -rf $diyhue.tar.gz && pip3 install -r /diyhue/requirements.txt --no-cache-dir' returned a non-zero code: 1

**Comment by DiegoJp** [comment]
Hi, I got the same error while trying to rebuild:

![imagen](https://github.com/diyhue/diyHue/assets/41808318/14d9b47c-0343-4933-8c30-e2a9125ea5c3)

**Comment by fanaticDavid** [comment]
Same here, I can't install diyHue 1.5.0 (it was going to be my first install).
Home Assistant 2023.7.3
Supervisor 2023.07.1
Operating System 10.3
on a Proxmox VM

**Comment by Killian3145** [comment]
I am also getting the same error, I am running Homeassistant 10.3 on a Virtualbox VM

**Comment by fanaticDavid** [comment]
While revisitting this issue, I noticed the repository I added in my HA instance was wrong?
I had the repository `https://github.com/MaxBec/diyHue-hassio` which I got from the HA community forum. With that repository, version 1.5.0 tries to install but fails with the error discussed above.

I just removed that repository and added this one: `https://github.com/diyhue/hassio-addon`
That successfully installed version 2.0.9 of diyHue, at least for me. Now I finally get to see what it has to offer.

Btw, as this issue concerns the add-on I'm guessing it should've been posted [here](https://github.com/diyhue/hassio-addon/issues).
